core/src/test/java/io/grpc/internal/TransportSetTest.java

```diff
@@ -80,7 +80,6 @@
   private static final String authority = "fakeauthority";
 
   private FakeClock fakeClock;
-  private FakeClock fakeExecutor;
```

Comment:
This was to verify that DelayedClientTransport creates the real streams on an executor that is not the schedulingExecutor (the fakeClock in the test).